apps/auction2/actions.js

```diff
@@ -0,0 +1,18 @@
+// Action types
+export const UPDATE_ARTWORKS = 'UPDATE_ARTWORKS'
+export const TOGGLE_LIST_VIEW = 'TOGGLE_LIST_VIEW'
+
+// Action creators
+export function updateArtworks(artworks) {
+  return {
+    type: UPDATE_ARTWORKS,
+    artworks
+  }
+}
+
+export function toggleListView(listView) {
+  return {
+    type: TOGGLE_LIST_VIEW,
+    listView
```

Comment:
One thing I would do is move the "payload" into a payload key, so that actions can conform to whats known as a "flux standard action", a convention that you see throughout the Redux world: https://github.com/acdlite/flux-standard-action.

```javascript
export function updateArtwork(artworks) {
  return {
    type: UPDATE_ARTWORKS, 
    payload: {
       artworks
    }
  }
}
```

This is a good way to keep different types of data that's going over the flux wire namespaced and isolated. For example, its very common to see a `meta` keyword used to perform various actions or to pass along related data: https://github.com/johanneslumpe/redux-history-transitions/blob/master/example/actions/Items.js#L19